content/en/docs/tasks/debug-application-cluster/debug-service.md

```diff
@@ -164,7 +164,7 @@ No resources found.
 Error from server (NotFound): services "hostnames" not found
 ```
 
-So we have a culprit, let's create the Service.  As before, this is for the
+So you have a culprit, let's create the Service.  As before, this is for the
```

Comment:
nit: Do you need, `So you have a culprit`,
Let's create the Service (explain how)